config.toml

```diff
@@ -548,6 +548,10 @@ logo = "vitc.png"
 url = "https://www.workiva.com"
 logo = "workiva.png"
 
+[[params.users]]
+url = "https://www.xbox.com"
```

Comment:
Thanks @tomkerkhove ! Happy to approve Xbox being listed as a KEDA end-user :)

Reply:
Thank you @christian-macedo !